src/common/AndroidUtils.ts

```diff
@@ -24,60 +24,15 @@ const spawn = childProcess.spawn;
 type StdioOptions = childProcess.StdioOptions;
 
 const LOGGER_NAME = 'force:lightning:mobile:android';
+const USER_HOME =
+    process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
+const WINDOWS_OS = 'win32';
+const ANDROID_SDK_MANAGER_NAME = 'sdkmanager';
+const ANDROID_AVD_MANAGER_NAME = 'avdmanager';
+const ANDROID_ADB_NAME = 'adb';
+const DEFAULT_ADB_CONSOLE_PORT = 5554;
 
 export class AndroidSDKUtils {
-    static get androidHome(): string {
-        return process.env.ANDROID_HOME ? process.env.ANDROID_HOME.trim() : '';
-    }
-    public static USER_HOME =
```

Comment:
Not sure why these were defined as public static. Almost all of them were only used internally in this file. So I refactored some as constants and the others as private internal variables/caches with private getters (which makes unit testing more straight forward).